InstallCob4.sh

```diff
@@ -154,12 +154,13 @@ function NFSSetup
   if [ "$MODE" == "master" ]
     then
       sudo apt-get install apt-cacher-ng
-      sudo echo "server 0.pool.ntp.org" | sudo tee -a /etc/ntp.conf
-      sudo echo "restrict $IP mask 255.255.255.0 nomodify notrap" | sudo tee -a /etc/ntp.conf
+      sudo cp -f ~/git/setup_cob4/cob-pcs/ntp_server /etc/ntp.conf
+      sudo sed -i "s/server_ip/$IP/g" /etc/ntp.conf
   elif [ "$MODE" == "slave" ]
     then
-      sudo echo "server $server" | sudo tee -a /etc/ntp.conf
-      grep -q -F 'Acquire::http::Proxy \""http://'$server':3142"\";' /etc/apt/apt.conf.d/01proxy || sudo sh -c '(echo "Acquire::http::Proxy \""http://'$server':3142"\";") > /etc/apt/apt.conf.d/01proxy'
+      sudo cp -f ~/git/setup_cob4/cob-pcs/ntp_client /etc/ntp.conf
```

Comment:
this has nothing to do with NFS-Setup. This is Apt-Cacher setup. @ipa-nhg Do you want to split that?